### DISCUSSION TITLE
How to restore appwrite sql dumps with utf-8 encoding

### DISCUSSION THREAD

**Discussion by L-Andy** [appwrite/appwrite#3679] [Q&A]
I was recently working on a project and i needed to backup the db(collection), but on trying to restore everything seemed to execute fine, but it nothing was written to the db and nothing as shown in console.
And after some inspections of the dump files one from a project without any french characters it was in ASCII encoding and the one with french characters was in UTF-8.

Does that mean appwrite doesn't support UTF-8 by default?

**Comment by tripolskypetr** [comment]
If you facing the 500 Internal Server Error while using AppWrite Cloud data migration, [that json-per-document migration tool](https://github.com/react-declarative/appwrite-backup-tool) will be quite useful
